From d504d45114ad96cc143bcc2107529b2bf544f248 Mon Sep 17 00:00:00 2001
From: Jan Beich <jbeich@FreeBSD.org>
Date: Sat, 9 Nov 2024 20:30:25 +0000
Subject: [PATCH] os: add missing header for BSDs after fd4be8b9ca93 (#16)

src/os/Process.cpp:105:9: error: use of undeclared identifier 'sigemptyset'
  105 |         sigemptyset(&set);
      |         ^
src/os/Process.cpp:106:21: error: use of undeclared identifier 'SIG_SETMASK'
  106 |         sigprocmask(SIG_SETMASK, &set, NULL);
      |                     ^
---
 src/os/Process.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/os/Process.cpp b/src/os/Process.cpp
index 4ffd900..2cf00ea 100644
--- src/os/Process.cpp
+++ src/os/Process.cpp
@@ -1,6 +1,7 @@
 #include <hyprutils/os/Process.hpp>
 using namespace Hyprutils::OS;
 
+#include <csignal>
 #include <cstdio>
 #include <unistd.h>
 #include <cstring>
